src/gei/Commands/GenerateScriptCommand.cs

```diff
@@ -111,6 +112,11 @@ public class GenerateScriptCommand : CommandBase<GenerateScriptCommandArgs, Gene
 
         public Option<bool> Verbose { get; } = new("--verbose");
 
+        public Option<bool> KeepArchive { get; } = new("--keep-archive")
+        {
+            Description = "Keeps the archive on the user machine after uploading to the storage account."
```

Comment:
One more thing from me - but otherwise I'm good! This would also apply to `MigrateRepoCommand`:

```suggestion
            Description = "Keeps the archive on this machine after uploading to the blob storage account. Only applicable for migrations from GitHub Enterprise Server versions before 3.8.0."
```

Reply:
Done :)